packages/core/src/config.ts

```diff
@@ -557,6 +565,23 @@ const composeFormatConfig = ({
 
       return config;
     }
+    case 'mf':
+      return {
+        tools: {
+          rspack: {
+            output: {
+              uniqueName: pkgJson.name as string,
+            },
+            // TODO when we provide dev mode for rslib mf format, this should be modified to as the same with config.mode
+            // can not set nodeEnv to false, because mf format should build shared module.
+            // If nodeEnv is false, the process.env.NODE_ENV in third-party packages's will not be replaced
+            // now we have not provide dev mode for users, so we can always set nodeEnv as 'production'
```

Comment:
>  now we have not provide dev mode for users

Could we consider `rslib build --watch` as the dev mode if `watch` is on?

Reply:
we consider to give a dev server like storybook to achieve this